docker/Dockerfile.umn

```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
     ros-humble-grid-map-rviz-plugin \
     ros-humble-can-msgs \
     ros-humble-usb-cam \
+    python3-pytest \
+    python3-opencv \
+    python3-serial \
+    python3-numpy \
+    libboost-all-dev \
+    ros-humble-ros-gz \
+    ros-humble-sdformat-urdf \
+    ros-humble-joint-state-publisher-gui \
+    python3-tqdm \
+    python3-requests \
+    libeigen3-dev \
+    ros-humble-librealsense2 \
```

Comment:
Why is this not in real sense layer?

Reply:
idk man, you tell me

Reply:
oh its prob because skimmer node depends on realsense

Reply:
Ohhhh, can it be decoupled :(

Reply:
eventually yeah, but it's probably fine for now